src/mme-app/actionHandlers/attachActionHandlers.cpp

```diff
@@ -38,11 +54,99 @@
 #include <utils/mmeCauseUtils.h>
 #include "gtpCauseTypes.h"
 
+
+
+#define MCC_MNC_LEN 4
+#define LB_HB_LEN 8
```

Comment:
what is this #define ?

Reply:
this is max length of MCC, MNC and lb,hb for tac based query